### PR TITLE
Storybook > Link

### PR DESCRIPTION
**Pull request by mckelvey**
This PR adds a link component, link stories, and updates buttons and links in existence; it fixes a couple of bugs in the process.

### CODE REVIEW

**Comment by mckelvey** [comment]
[Disabled Button Pointer](https://trello.com/c/ERfwXFbG/27-disabled-button-pointer)

**Comment by mckelvey** [comment]
[Add Link Hover / Active states](https://trello.com/c/ZQ2oPBKy/28-add-link-hover-active-states)

File: components/ui/Link.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import styled, { css } from 'styled-components'
+
+import { linkStyles } from './Typography'
+import theme from '../../util/theme'
+
+const sharedLinkStyles = css`
+  ${linkStyles}
+  justify-self: left;
+  margin: 0 auto 0 0;
+  cursor: ${props => props.theme.cursors.select};
```
Comment (mackermedia) [inline components/ui/Link.js:12]: why call this style `select` and not `pointer`?

**Comment by mackermedia** [comment]
Looks good! 👍

File: components/ui/Link.js (hunk 1 above)
Comment (mckelvey) [reply components/ui/Link.js:12]: This is a good point. :)

**Pull request merged**